### DISCUSSION TITLE
Realearn feedback on Native Instruments Maschine MK3

### DISCUSSION THREAD

**Discussion by AsVit** [helgoboss/helgobox#894] [Q&A] [answered]
Hi. Is it possible to display channel or master track volume, or sends level in dB on the MK3 screen, instead of just 0-127? (vol 88 on maschine screen photo = -2.96 dB in reaper) 

Maschine MK3 screen:
![photo_2023-11-01_14-59-12](https://github.com/helgoboss/realearn/assets/13093091/4be123cf-6375-46f0-b3a6-c91a15f5289f)

My current settings for volume:
<img width="503" alt="Screenshot 2023-11-01 153745" src="https://github.com/helgoboss/realearn/assets/13093091/068d4739-6141-4cd4-b82a-34c5716eb960">

**Comment by helgoboss** [comment, marked as answer]
No, not possible. There's no specific support for Maschine built into ReaLearn.

I'm not sure if Maschine supports sysex, then you could program it yourself via MIDI Script source, but even then it would require much knowledge on your part.

**Reply by AsVit** [reply]
Thank you!
